core/PhysiCell_cell_container.cpp

```diff
@@ -124,6 +124,14 @@ void Cell_Container::update_all_cells(double t, double phenotype_dt_ , double me
 {
 	// secretions and uptakes. Syncing with BioFVM is automated. 
 
+	static double time_since_last_cycle = phenotype_dt_;
```

Comment:
Awesome! I like this new structure—it's clearer and gets rid of the initialized test. Consider renaming the variable `time_since_last_cycle` to `time_since_last_phenotype` for consistency.

Reply:
Thanks! And done!